modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing an upstream vSphere CSI Operator Driver
+
+If you have installed a non-Red Hat vSphere Container Storage Interface (CSI) Operator Driver, and then try to update your cluster to {product-title} 4.11 or later, {product-title} prevents the update.
+
```

Comment:
How about something like:

"Openshift 4.10 comes with a built-in version of vSphere CSI driver and hence after upgrading to 4.10, your cluster may have upgrades to next major version(such as 4.11) of Openshift disabled - if there is a non-redhat provided vSphere CSI driver present in the cluster. The 4.10 cluster is still fully supported and upgrades to bug fix versions of 4.10(4.10.z) are not blocked but you must correct this state before upgrades to next major version of Openshift can be enabled"

Reply:
I know according to semver - 4.11 is basically a minor version but I don't know precise language we use in openshift-docs to convey that meaning. I don't expect users reading these docs to be familiar with semver (or other nuances of versioning).